src/Event/BlockEvent.php

```diff
@@ -14,12 +14,9 @@
 namespace Sonata\BlockBundle\Event;
 
 use Sonata\BlockBundle\Model\BlockInterface;
-use Symfony\Contracts\EventDispatcher\Event;
+use Symfony\Component\EventDispatcher\Event;
```

Comment:
:thinking: how does this relate to closing the API?

Reply:
fixed